html/say-what-admin-addedit.php

```diff
@@ -15,11 +15,11 @@
 			<input type="text" name="say_what_orig_string" size="120" value="<?php echo esc_attr( $replacement->orig_string ) ?>"><br/>
 		</p>
 		<p>
-			<label for="say_what_domain"><?php _e( 'Text domain', 'say_what' ); ?></label> <a href="http://plugins.leewillis.co.uk/doc_post/adding-string-replacement/"><i class="dashicons dashicons-info">&nbsp;</i></a><br/>
+			<label for="say_what_domain"><?php _e( 'Text domain', 'say_what' ); ?></label> <a href="http://plugins.leewillis.co.uk/doc_post/adding-string-replacement/" target="_blank" tabindex="-1"><i class="dashicons dashicons-info">&nbsp;</i></a><br/>
 			<input type="text" name="say_what_domain" size="30" value="<?php echo esc_attr( $replacement->domain ); ?>"><br/>
 		</p>
 		<p>
-			<label for="say_what_context"><?php _e( 'Text context', 'say_what' ); ?></label> <a href="http://plugins.leewillis.co.uk/doc_post/replacing-wordpress-strings-context/"><i class="dashicons dashicons-info">&nbsp;</i></a><br/>
+			<label for="say_what_context"><?php _e( 'Text context', 'say_what' ); ?></label> <a href="http://plugins.leewillis.co.uk/doc_post/replacing-wordpress-strings-context/" target="_blank" tabindex="-1"><i class="dashicons dashicons-info">&nbsp;</i></a><br/>
```

Comment:
My intention was to get those out of sequential tabbing, so that we can use
tab to navigate smoothly between the text fields. But you're right, I did
not give enough consideration to the accessibility issues...

On Sat, Mar 28, 2015 at 3:46 PM, Lee Willis <notifications@github.com>
wrote:

> In html/say-what-admin-addedit.php
> <https://github.com/leewillis77/say-what/pull/12#discussion_r27343190>:
>
> >  			<input type="text" name="say_what_domain" size="30" value="<?php echo esc_attr( $replacement->domain ); ?>"><br/>
> >  		</p>
> >  		<p>
> > -			<label for="say_what_context"><?php _e( 'Text context', 'say_what' ); ?></label> <a href="http://plugins.leewillis.co.uk/doc_post/replacing-wordpress-strings-context/"><i class="dashicons dashicons-info">&nbsp;</i></a><br/>
> > +			<label for="say_what_context"><?php _e( 'Text context', 'say_what' ); ?></label> <a href="http://plugins.leewillis.co.uk/doc_post/replacing-wordpress-strings-context/" target="_blank" tabindex="-1"><i class="dashicons dashicons-info">&nbsp;</i></a><br/>
>
> Same here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/leewillis77/say-what/pull/12/files#r27343190>.
>